### PR TITLE
Apple clone is added

### LINKED ISSUE #2231: Apple-Clone
**Describe the project you want to add with tech stack**

Apple-Clone with  html, Css 

Assign this issue to me under GSSOC'23

### PR DESCRIPTION
**Pull request by shraddha761**
# Title and Issue number 

Close #2231 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by shraddha761** [comment]
@MohitGupta121 @pranjay-poddar please review

File: Frontend-Projects/Apple-Clone/images/img1.jpeg
Comment (MohitGupta121) [inline Frontend-Projects/Apple-Clone/images/img1.jpeg:1]: Remove images from PR and use them through link of images upload anywhere in cloud and fetch them.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
Same for all images.

**Comment by shraddha761** [comment]
@TusharKesarwani please review

**Review by TusharKesarwani — APPROVED** [review]

File: Frontend-Projects/Apple-Clone/images/img1.jpeg
Comment (TusharKesarwani) [reply Frontend-Projects/Apple-Clone/images/img1.jpeg:1]: I think images are of less size
So it wouldn't affect much

**Pull request closed without merging**